### DISCUSSION TITLE
Is there an official way to set a fixed IP per VM?

### DISCUSSION THREAD

**Discussion by felixstoessel** [utmapp/UTM#5072] [UTM Help (macOS)] [answered]
**Describe the issue**  
I want to use a Debian VM for local development on my Mac. This VM has NGINX as a web server, so I could open my website on my host system in the browser and it directs it to the guest system. I can not get this to work without the use of 127.0.0.1 and using different ports. Is there a way to set a fixed IP for my VM like `--ip=192.168.22.22`? So I can have a nice /etc/hosts on my host system like this:

```
192.168.22.22 new-project.test
```

**Configuration**  
* UTM Version: Version 4.1.5 (74)
* macOS Version: 12.6.1 (21G217)
* Mac Chip (Intel, M1, ...): M1

**Comment by osy** [comment, marked as answer]
Currently not possible with NAT, but with emulated VLAN, you can port forward to your host machine.

**Comment by Makr91** [comment]
You can also add a bridged network to your UTM server that's using the same NIC that your host machine is using to connect to your Home LAN.

If you have a DHCP server on your Home LAN, it should give it a IP that you can then access in a browser. (and then even set a /etc/hosts record to it)

To make it permanent, either set it statically in the VM, or assign it via DHCP reservation in your Routers.
